apps/src/sites/studio/pages/googleBlocklyWrapper.js

```diff
@@ -21,6 +21,15 @@ import CdoWorkspaceSvg from '@cdo/apps/blocklyAddons/cdoWorkspaceSvg';
 const BlocklyWrapper = function(blocklyInstance) {
   this.version = BlocklyVersion.GOOGLE;
   this.blockly_ = blocklyInstance;
+
+  /**
+   * If an id is not specified, the Block constructor will randomly generate a 20-character
+   * uid string. In UI tests, we want to be able to reference blocks by their id, which won't
```

Comment:
It sounds like this is no longer just for UI tests but also for runtime behaviour for block highlighting?

Reply:
Well I could also just update the regex in StudioApp if we wanted.. The only place that has a firm requirement for these IDs to be predictable is the UI tests. But we do reference blocks by their id throughout runtime code as well as tests.